vllm/distributed/kv_transfer/kv_connector/v1/offloading_connector.py

```diff
@@ -429,15 +573,23 @@ def register_kv_caches(self, kv_caches: dict[str, torch.Tensor]):
     def start_load_kv(self, metadata: OffloadingConnectorMetadata):
         for req_id, transfer_spec in metadata.reqs_to_load.items():
             job_id = self._generate_job_id()
-            self._jobs[job_id] = (req_id, False)
+            current_time = time.perf_counter()
+            src_spec, dst_spec = transfer_spec
+            assert isinstance(src_spec, BlockIDsLoadStoreSpec)
+            num_blocks = len(src_spec.block_ids)
+            self._jobs[job_id] = (req_id, False, current_time, num_blocks)
```

Comment:
I would prefer if we add `LoadStoreSpec.size_bytes()`, and then track size in bytes instead of num_blocks.